app/grandchallenge/cases/tasks.py

```diff
@@ -543,6 +546,8 @@ def import_dicom_to_healthimaging(*, dicom_imageset_upload_pk):
         upload.start_dicom_import_job()
     except RetryStep:
         raise
+    except RejectedDICOMFileError as e:
+        upload._mark_failed(error_message=e.justification)
```

Comment:
Chris confirmed that the justifications for these are all save to return to the user.

Reply:
See https://github.com/DIAGNijmegen/rse-roadmap/issues/419#issuecomment-3274109165